src/components/views/Articles/Articles.scss

```diff
@@ -4,6 +4,14 @@
 .article-body {
   @include marg(90px 0);
 
+  &:first-child {
+    @include marg(0 0 90px 0);
```

Comment:
Do you really need such simple mixins? Why is it better than `margin: x x x x;`?